queue_job/fields.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# copyright 2016 Camptocamp
+# license agpl-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+import json
+
+from odoo import fields, models
+
+
+class JobSerialized(fields.Field):
+    """ Serialized fields provide the storage for sparse fields. """
+    type = 'serialized'
```

Comment:
@guewen you must chose and other type name since this value is already used by the `fields.Serialized` in  Odoo. https://github.com/odoo/odoo/blob/10.0/odoo/fields.py#L2372
It's important to put a discriminant value for the type since it is used as key to register your field in the field's registry https://github.com/odoo/odoo/blob/10.0/odoo/fields.py#L104. This registry is used for example to fill the selection list when you add a new field with the UI....